lib/src/replaced_element.dart

```diff
@@ -96,6 +96,7 @@ class IframeContentElement extends ReplacedElement {
   final double width;
   final double height;
   final NavigationDelegate navigationDelegate;
+  final GlobalKey key = GlobalKey();
```

Comment:
Does it have to be a global key? Pretty sure this can be an ObjectKey.

Reply:
Sure it could be any sort of `Key`. Is there an advantage to using one type over another?

Reply:
Yes, for efficiency.

Reply:
@erickok changed the key to `UniqueKey`, its essentially the same as `ObjectKey` but doesn't require an `Object` argument.